pulsar-functions/proto/src/main/proto/Function.proto

```diff
@@ -70,6 +70,7 @@ message SourceSpec {
     SubscriptionType subscriptionType = 3;
     map<string,string> topicsToSerDeClassName = 4;
     uint64 timeoutMs = 6;
+    string topicsPattern = 7;
```

Comment:
Also where is the equivalent in the functionConfig

Reply:
yes, I have added e2e test in #1904 with admin-api integration. but missed to add into `CmdSink`. will fix it.

Reply:
fixed it.